python-api/src/main/python/api/test_main.py

```diff
@@ -0,0 +1,27 @@
+#
+# Licensed to Cloudera, Inc. under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Cloudera, Inc. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# THIS IS A TEST CLASS AND WILL BE REMOVED BEFORE MOVING THE LIBRARY TO PROD
```

Comment:
You should think about how to do unit testing for this. That means that, like in the java case, you'll have a `test` directory under `src`, and you'll need to hook it up to the maven execution (http://www.mojohaus.org/exec-maven-plugin/ might be helpful).